src/magaz/WebsiteBundle/Controller/IndexController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace magaz\WebsiteBundle\Controller;
+
+use magaz\CommonBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+//use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template
+
+
+//IndexController наследует пока  класс    Controller
+//Пока не знаю  как  реализовать  'WebsiteBundle:tamplate:index.html.twig'  через  AbstractController
+ 
+class IndexController extends Controller
```

Comment:
As parent for all your controllers use `AbstractController` from `CommonBundle`, don't use `Controller`!!!